source/developer/developer-setup.md

```diff
@@ -82,6 +82,10 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+_TIP_: There is a package in the [Arch User Repository](https://aur.archlinux.org/) called [Mattermost](https://aur.archlinux.org/packages/mattermost/). Follow the [AUR instructions](https://wiki.archlinux.org/index.php/Arch_User_Repository) to build and install. 
```

Comment:
I made a new PR  with more instructions as suggested: Update developer-setup.md #685

Sorry but I am not a github PR geek and couldn't find how to keep on this one.